testsuite/tests/typing-objects/pr6383.ml.reference

```diff
@@ -1,6 +1,6 @@
 
-# Characters 10-16:
+# Characters 11-16:
   let f (x: #M.foo) = 0;;
-            ^^^^^^
+             ^^^^^
```

Comment:
Note: it's not possible to refine the location here without adding more location information inside `Longident.t`, which would result in a painful change to the interface used by everyone manipulating the parsetree -- not a good idea.